docs/source/pretrained_models.rst

```diff
@@ -10,3 +10,17 @@ For convenience, the following pre-trained models are ready-to-use:
 - `findcord_tumor <https://github.com/ivadomed/findcord_tumor/archive/r20200621.zip>`_: Cord localisation model, trained on T2-weighted images with tumor.
 - `model_find_disc_t1 <https://github.com/ivadomed/model_find_disc_t1/archive/r20201013.zip>`_: Intervertebral disc detection model trained on T1-weighted images.
 - `model_find_disc_t2 <https://github.com/ivadomed/model_find_disc_t2/archive/r20200928.zip>`_: Intervertebral disc detection model trained on T2-weighted images.
+
+Packaged model format
+=====================
+Each folder contains a model (.pt or .onnx) with its corresponding configuration file. The packaged model is
+automatically generated during training. The folder containing the packaged model will be saved at the path specified by
+the key "log_directory" in the configuration file. The packaged model, the configuration file, and the model file will
```

Comment:
```suggestion
the key "log_directory/model_name" in the configuration file.
```

Reply:
Since there are two parameters (one for the log_directory and one for model_name) I feel this way is a bit confusing?